_sass/_layouts--global.scss

```diff
@@ -11,3 +11,12 @@ body {
     flex: 1;
   }
 }
+
+body {
+  // left the gap for header navbar
+  padding-top: $size--header-height;
```

Comment:
But every page has the header @TheMightyPenguin the idea is to make sure every page has that space on top so the header won't be placed over any content on the top of the page. Makes sense?